### PR TITLE
Create a changelog to document changes between versions

### LINKED ISSUE #113: Add ChangeLog to repository
Create a `CHANGELOG.md` that documents changes between versions.

### PR DESCRIPTION
**Pull request by evantorrie**
fixes #113

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+- `macaron` trace instrumentation (#20)
+- `dogstatsd` metrics exporter (#10)
+- `datadog` metrics exporter (#22)
+- Tags to all modules in repository
+- Repository folder structure and automated build (#3)
+
+### Changes
+
+- Prefix support for dogstatsd (#34)
+- Update Go Runtime package to use batch observer (#44)
```
Comment (MrAlias) [inline CHANGELOG.md:65]: Missing links.

```suggestion
- Update Go Runtime package to use batch observer (#44)

[Unreleased]: https://github.com/open-telemetry/opentelemetry-go-contrib/compare/v0.7.0...HEAD
[0.7.0]: https://github.com/open-telemetry/opentelemetry-go-contrib/releases/tag/v0.7.0
[0.6.1]: https://github.com/open-telemetry/opentelemetry-go-contrib/releases/tag/v0.6.1
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- Bump github.com/golangci/golangci-lint from 1.21.0 to 1.27.0 in /tools (#77)
+- Bump go.mongodb.org/mongo-driver from 1.3.2 to 1.3.4 in /instrumentation/go.mongodb.org/mongo-driver (#76)
+- Bump github.com/stretchr/testify from 1.5.1 to 1.6.1 (#74)
+- Bump gopkg.in/macaron.v1 from 1.3.5 to 1.3.9 in /instrumentation/macaron (#68)
+- Bump github.com/gin-gonic/gin from 1.6.2 to 1.6.3 in /instrumentation/gin-gonic/gin (#73)
+- Bump github.com/DataDog/datadog-go from 3.5.0+incompatible to 3.7.2+incompatible in /exporters/metric/datadog (#78)
+
+### Removed
+
+- `internal/trace/http.go` helpers, replaced by `api/standard` helpers in otel-go repo (#112)
```
Comment (MrAlias) [inline CHANGELOG.md:43]: Could probably move this to `Changed` section as it doesn't remove a feature exposed to the end user, rather changes underlying implementation.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+## [0.7.0] - 2020-06-29
+
+This release upgrades its [go.opentelemetry.io/otel](https://github.com/open-telemetry/opentelemetry-go "Otel Github") dependency to v0.7.0.
+
+### Added
+
+- Create `RELEASING.md` instructions (#101)
+- Apply transitive dependabot go.mod updates as part of a new automatic Github workflow. (#94)
+- New dependabot integration to automate package upgrades. (#61)
+- Add automatic tag generation script for release (#60)
```
Comment (MrAlias) [inline CHANGELOG.md:25]: +1 on being consistent with sentence fragments or full sentences.

```suggestion
- Create `RELEASING.md` instructions. (#101)
- Apply transitive dependabot go.mod updates as part of a new automatic Github workflow. (#94)
- New dependabot integration to automate package upgrades. (#61)
- Add automatic tag generation script for release. (#60)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+- Added output validation to Datadog example (#96)
+- Move Macaron package to match layout guidelines. (#92)
+- Update top-level README and instrumentation README. (#92)
+- Bump google.golang.org/grpc from 1.29.1 to 1.30.0 (#99)
+- Bump github.com/golangci/golangci-lint from 1.21.0 to 1.27.0 in /tools (#77)
+- Bump go.mongodb.org/mongo-driver from 1.3.2 to 1.3.4 in /instrumentation/go.mongodb.org/mongo-driver (#76)
+- Bump github.com/stretchr/testify from 1.5.1 to 1.6.1 (#74)
+- Bump gopkg.in/macaron.v1 from 1.3.5 to 1.3.9 in /instrumentation/macaron (#68)
+- Bump github.com/gin-gonic/gin from 1.6.2 to 1.6.3 in /instrumentation/gin-gonic/gin (#73)
+- Bump github.com/DataDog/datadog-go from 3.5.0+incompatible to 3.7.2+incompatible in /exporters/metric/datadog (#78)
```
Comment (MrAlias) [inline CHANGELOG.md:39]: +1 on being consistent with sentence fragments or full sentences.
```suggestion
- Upgrade Datadog metrics exporter to include Resource tags. (#46)
- Added output validation to Datadog example. (#96)
- Move Macaron package to match layout guidelines. (#92)
- Update top-level README and instrumentation README. (#92)
- Bump google.golang.org/grpc from 1.29.1 to 1.30.0. (#99)
- Bump github.com/golangci/golangci-lint from 1.21.0 to 1.27.0 in /tools. (#77)
- Bump go.mongodb.org/mongo-driver from 1.3.2 to 1.3.4 in /instrumentation/go.mongodb.org/mongo-driver. (#76)
- Bump github.com/stretchr/testify from 1.5.1 to 1.6.1. (#74)
- Bump gopkg.in/macaron.v1 from 1.3.5 to 1.3.9 in /instrumentation/macaron. (#68)
- Bump github.com/gin-gonic/gin from 1.6.2 to 1.6.3 in /instrumentation/gin-gonic/gin. (#73)
- Bump github.com/DataDog/datadog-go from 3.5.0+incompatible to 3.7.2+incompatible in /exporters/metric/datadog. (#78)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Changed
+
+- Update `CONTRIBUTING.md` to ask for updates to `CHANGELOG.md` with each pull request. (#114)
+- Create this `CHANGELOG.md` (#114)
```
Comment (MrAlias) [inline CHANGELOG.md:14]:
```suggestion
- Create this `CHANGELOG.md`. (#114)
```

File: CHANGELOG.md (hunk 2 above)
Comment (evantorrie) [reply CHANGELOG.md]: Done

**Pull request merged**